### PR TITLE
adding logging_tree recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: https://github.com/nsls-ii-forge/logging_tree-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

**Review by mrakitin — APPROVED** [review]
@kbeeperez, looks good to me.
@conda-forge/staged-recipes, I agree to be a maintainer of this package.

**Comment by mrakitin** [comment]
@brandon-rhodes, are you willing to co-maintain the conda-forge feedstock for your package?

File: recipes/logging_tree/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "logging_tree" %}
+{% set version = "1.9" %}
+{% set sha256 = "fe78b28788c249b515d12a50417e4a96ba095582927b3601a0573886bffc1ff0" %}
```
Comment (ocefpaf) [inline recipes/logging_tree/meta.yaml:3]: You don't need a jinja variable that is used only once. The recipe is more readable if this goes into line 11.

**Comment by mrakitin** [comment]
Thanks for the review/merge, @ocefpaf!

**Comment by brandon-rhodes** [comment]
@mrakitin — I'm not sure I should be taking on more responsibilities at this point, given other commitments. I'll be happy to answer any questions if you have problems packaging new releases, though.

**Comment by mrakitin** [comment]
Thank you for the reply, @brandon-rhodes! We'll try to handle it, and I'll get back to you in case of issues :)

On a side note, I watched a few of your videos on Youtube about Python and other things, they are great!

**Comment by brandon-rhodes** [comment]
Thanks for the kind words, and feel free to @ - mention me later if the conda package runs into trouble.

**Pull request merged**